app/root.tsx

```diff
@@ -109,6 +127,20 @@ export function CatchBoundary() {
         <meta name="viewport" content="width=device-width,initial-scale=1" />
         <Meta />
         <Links />
+        <script
+          async
+          id="gtag-init"
+          dangerouslySetInnerHTML={{
+            __html: `
+                window.dataLayer = window.dataLayer || [];
+                function gtag(){dataLayer.push(gaTrackingId);}
+                gtag('js', new Date());
+                gtag('config', '${gaTrackingId}', {
```

Comment:
<img width="682" alt="Screenshot 2023-03-08 at 19 13 36" src="https://user-images.githubusercontent.com/88032788/223796057-d6350c80-2c7b-44df-811e-dab4880c50de.png">

My VS code says this value is out of reach, it seems it's local to the other function, maybe we need to use `const { gaTrackingId } = useLoaderData()` in this scope as well?

Reply:
Out of curiosity, why is it in `CatchBoundary` function? I would have put it on the `Head` in the function above, on the line 87